testsuite/features/step_definitions/command_steps.rb

```diff
@@ -834,7 +834,14 @@
   filename = File.basename(path)
   cmd = "configure-proxy.sh --non-interactive --rhn-user=admin --rhn-password=admin --answer-file=#{filename}"
   proxy_timeout = 600
-  $proxy.run(cmd, true, proxy_timeout, 'root')
+  return_code = 0
+  # Workaround issue https://github.com/SUSE/spacewalk/issues/9165
+  begin
+    return_code = $proxy.run(cmd, true, proxy_timeout, 'root')
+  rescue StandardError => e
+    puts "Catched exception (see bsc#1138952): #{e}"
```

Comment:
ok, nice, thanks

Reply:
It could be a good idea to tag this with the following comment:
`# WORKAROUND bsc#1138952`